<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>730</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1030</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рабочее место оператора стенда</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxPorts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>ПОДКЛЮЧЕНИЕ</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_port">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ports">
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_connectPort">
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Подключить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_portStatus">
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Нет соединения</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>500</width>
      <height>570</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTest">
     <attribute name="title">
      <string>Испытания</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_Atch">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>480</width>
        <height>160</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string notr="true">АЧТ</string>
      </property>
      <widget class="QCheckBox" name="checkBoxVentilator">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>37</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Включить вентилятор</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxChT">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>37</y>
         <width>135</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Включить АЧТ</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFonTemp">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>73</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Текущая дельта, град.</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditFonTemp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>73</y>
         <width>100</width>
         <height>34</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDeltaTemp">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>115</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelDeltaTemp">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>115</y>
         <width>210</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Задать дельту, град.</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_ports_2">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>115</y>
         <width>100</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_Izluch">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>175</y>
        <width>480</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string notr="true">ИЗЛУЧАТЕЛИ</string>
      </property>
      <widget class="QCheckBox" name="checkBoxIzluch1">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>60</y>
         <width>163</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>ВКЛ/ВЫКЛ излуч. №1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxIzluch2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>100</y>
         <width>164</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>ВКЛ/ВЫКЛ излуч. №2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonIzluch1Down">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>50</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Тусклее</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonIzluch2Down">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>90</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Тусклее</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonIzluch1Up">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>50</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Ярче</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonIzluch2Up">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>90</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Ярче</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_Mira">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>334</y>
        <width>480</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string notr="true">ПОЗИЦИОНИРОВАНИЕ МИРЫ</string>
      </property>
      <widget class="QLabel" name="labelZadPolozh">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>81</y>
         <width>188</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Задать положение, мм</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditTekPolozh">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>40</y>
         <width>100</width>
         <height>34</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelTekuschPolozh">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>44</y>
         <width>179</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Текущее положение, мм</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelKorrecMira">
       <property name="geometry">
        <rect>
         <x>371</x>
         <y>21</y>
         <width>93</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>КОРРЕКЦИЯ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonKorrPlus">
       <property name="geometry">
        <rect>
         <x>405</x>
         <y>40</y>
         <width>60</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonKorrMinus">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>40</y>
         <width>60</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditZadPolozh">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>82</y>
         <width>100</width>
         <height>34</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxMiraZero">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>160</y>
         <width>155</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Запомнить &quot;нуль&quot;</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxResetMiraZero">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>155</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arimo</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Сбросить &quot;нуль&quot;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonMiraPosZad">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>80</y>
         <width>130</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>500</width>
      <height>550</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>МОДУЛЬ РАБОТЫ С КАМЕРОЙ</string>
    </property>
    <widget class="QComboBox" name="comboBox_cameras">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>37</y>
       <width>125</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arimo</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_connectCamera">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>36</y>
       <width>116</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arimo</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_camera">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>36</y>
       <width>80</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Камера</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cameraStatus">
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>39</y>
       <width>119</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arimo</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Нет соединения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_videoStream">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>420</width>
       <height>420</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditDebugPrintout">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>599</y>
      <width>500</width>
      <height>71</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QLabel" name="label_debugPrint">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>570</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отладочные сообщения</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
